### PR TITLE
View form

### LINKED ISSUE #12: As an User, I am able to view the form via link.
## Description
As an User, I am able to view the form via link.

## Mocks
n/a

### Update [Required]
- [ ] Page to view the form questions
- [ ] Space to answer the listed questions
- [ ] Button to submit the form
- [ ] Add checks for form close or required questions

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
2-3 days

### PR DESCRIPTION
**Pull request by bismitaguha**
### Description

Fixes #12 

### Type of Change:

- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

Manual
Screenshot from 2020-07-19 02-34-47

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by bismitaguha** [comment]
@sidvenu @abha224 only the last [commit](https://github.com/anitab-org/open-source-programs-web/pull/43/commits/5f100719de05e05f0731c9be26f1395bb6b807c1) is relevant to this PR.

File: src/components/Questions.js [hunk 1]
```diff
@@ -153,7 +161,7 @@ class Questions extends Component {
         const { questions, form, userinfo } = this.props
         const type = userinfo ? ( userinfo[0] ? userinfo[0].user_type : null ) : 'student'
         this.state.fields = questions
-        console.log(this.state.newfields)
+        console.log(form)
```
Comment (sidvenu) [inline src/components/Questions.js]: To be removed?

**Review by sidvenu — CHANGES_REQUESTED** [review]

**Review by sidvenu — DISMISSED** [review]
LGTM.

**Comment by sidvenu** [comment]
@bismitaguha may I know the updates on this PR?

**Comment by bismitaguha** [comment]
@sidvenu I had updated this PR in the beginning of last week only. Please review!

File: src/components/Questions.js [hunk 2]
```diff
@@ -327,9 +343,14 @@ class Questions extends Component {
...
 
-                {/* editable format of fields */}
+                {/* to enable preview feature */}
                 {
-                    form && form.length !== 0 ?
+                    ((this.state.preview && type === 'admin') || type !== 'admin') && (form && form.length !== 0) ?
+                    form.published_status !== 'closed' || (form.published_status === 'closed' && type === 'admin') ?
+                    <Preview id={this.props.match.params.id} />
+                    : <span>Sorry! This form is no longer accepting any responses.</span>
+                    :
```
Comment (sidvenu) [inline src/components/Questions.js:356]: This nested ternary operator needs to be refactored for better readability. If that isn't possible - add a comment.

**Review by sidvenu — CHANGES_REQUESTED** [review]

**Review by sidvenu — APPROVED** [review]
LGTM!

**Review by abha224 — APPROVED** [review]
This pr is reviewed and tested. The code is working as expected :+1:

**Pull request merged**